src/main/java/com/zufar/onlinestore/payment/converter/StripePaymentIntentConverter.java

```diff
@@ -0,0 +1,30 @@
+package com.zufar.onlinestore.payment.converter;
+
+import com.stripe.model.PaymentIntent;
+import com.stripe.model.PaymentMethod;
+import com.stripe.param.PaymentIntentCreateParams;
+import com.zufar.onlinestore.cart.dto.ShoppingSessionDto;
+import com.zufar.onlinestore.payment.calculator.PaymentPriceCalculator;
+import com.zufar.onlinestore.payment.entity.Payment;;
+import org.mapstruct.BeanMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingConstants;
+
+@Mapper(uses = PaymentPriceCalculator.class, componentModel = MappingConstants.ComponentModel.SPRING)
```

Comment:
maybe better use "spring" in the componentModel instead of MappingConstants.ComponentModel.SPRING ?